loadcell loadcell1
loadcell loadcell2

